pkg/services/ngalert/api/authorization.go

```diff
@@ -173,6 +173,9 @@ func (api *API) authorize(method, path string) web.Handler {
 	case http.MethodPost + "/api/alertmanager/grafana/config/api/v1/receivers/test":
 		fallback = middleware.ReqEditorRole
 		eval = ac.EvalPermission(ac.ActionAlertingNotificationsRead)
+	case http.MethodPost + "/api/alertmanager/grafana/config/api/v1/templates/test":
+		fallback = middleware.ReqEditorRole
```

Comment:
I wonder if it can be just Viewer. btw, default fallback for POST is Editor, so you do not need to change it.

Reply:
> I wonder if it can be just Viewer

I can't think of a good reason for it to fallback to Editor other than that the intended use is for editing templates. It can be less restrictive I think, the result of hitting the endpoint is not as impactful as making a request to an external service.